mlflow/tracing/processor/mlflow.py

```diff
@@ -116,6 +117,7 @@ def _start_trace(self, span: OTelSpan) -> TraceInfo:
                 request_id = encode_trace_id(span.context.trace_id)
                 trace_info = self._create_trace_info(request_id, span, experiment_id, metadata)
 
+        trace_info.tags.update(resolve_tags())
```

Comment:
Instead of updating here, I think we should pass tags=resolve_tags() to _create_trace_info and _start_tracked_trace

Reply:
what would be the benefit there?

Reply:
I think this line only affects in-memory trace_info, tags won't be saved in the backend

Reply:
Ahh I see, yeah I guess with what I'm doing here, the tags only get persisted when the trace ends. Let me take a look